### ISSUE TITLE
name agrument not passed correctly

### ISSUE THREAD

**Issue by hstejas** [Bailador/Bailador#265]
> This is Rakudo version 2017.09 built on MoarVM version 2017.09.1
> implementing Perl 6.c.

> Info for: zef
> Identity: zef:ver<0.1.29>:auth<github:ugexe>

OS: Ubuntu 17.04

This is on a fresh install of perl6 and bailador (tried manually building rakudo and using rakudobrew)

    bailador --name=test new

gives this error
```
 --name=Project-Name is a required parameter
```

whereas directly calling 

    rakudo/install/share/perl6/site/resources/F3147CDB5FA175F21CCAC05B6B29A24EF5EAED7B --name=test new

works fine

This diff points to the problem (./rakudo/install/share/perl6/site/bin/bailador)
```
@@ -1,4 +1,5 @@
 #!/usr/bin/env perl6
 sub MAIN(:$name is copy, :$auth, :$ver, *@, *%) {
-    CompUnit::RepositoryRegistry.run-script("bailador", :dist-name<Bailador>, :$name, :$auth, :$ver);
+       my $x = "--name=" ~ $name;
+    CompUnit::RepositoryRegistry.run-script("bailador", :dist-name<Bailador>, :$x, :$auth, :$ver);
 }
```
not sure if it is a zef issue or a bailador issue

**Comment by ufobat** [comment]
this script is not part of Bailador.

**Comment by szabgab** [comment]
Isn't this the script that is generated by zef when we install Bailador?

**Comment by ufobat** [comment]
https://irclog.perlgeek.de/perl6/2017-09-30#i_15238640 fyi

**Comment by ufobat** [comment]
=> we shoudn't use --name as a paremeter.
no --name, no --auth, no --ver

**Comment by ufobat** [comment]
will be fixed in 0.0.13
